src/main/java/decodes/cwms/CwmsSqlDatabaseIO.java

```diff
@@ -179,114 +193,113 @@ public void connectToDatabase(String sqlDbLocation)
+					int decodesDbVersion = getDecodesDatabaseVersion();
+					Logger.instance().info(module + " decodesDbVersion=" + decodesDbVersion);
+					keyGenerator = decodesDbVersion >= DecodesDatabaseVersion.DECODES_DB_68 ?
+							new OracleSequenceKeyGenerator() :
+							new CwmsSequenceKeyGenerator(decodesDbVersion);
+						TimeSeriesDb.readVersionInfo(this, conInfo.getConnection());
+						cgl.setLoginSuccess(true);
 
-			q = "ALTER SESSION SET nls_timestamp_tz_format = 'yyyy-mm-dd hh24:mi:ss'";
-			Logger.instance().info(q);
-			st.execute(q);
-			
-			Logger.instance().info("DECODES IF Connected to TSDB Version " + tsdbVersion);
-		}
-		catch(SQLException ex)
-		{
-			String msg = "Error in '" + q + "': " + ex
-				+ " -- will proceed anyway.";
-			Logger.instance().failure(msg + " " + ex);
-		}
-		finally
-		{
-			try { st.close(); } catch(Exception ex) {}
-		}
+					String q = null;
+					try(Statement st = conn.createStatement();)
+					{
+						// Hard-code date & timestamp format for reads. Always use GMT.
+						q = "ALTER SESSION SET TIME_ZONE = 'GMT'";
+						Logger.instance().info(q);
+						st.execute(q);
 
-		cgl.setLoginSuccess(true);
-		Logger.instance().info(module + 
-			" Connected to DECODES CWMS Database " + sqlDbLocation + " as user " + _dbUser
-			+ " with officeID=" + dbOfficeId + " (dbOfficeCode=" + conInfo.getDbOfficeCode() + ")");
-		
-		// CWMS-8979 Allow settings in the database to override values in user.properties.
-		String settingsApp = System.getProperty("SETTINGS");
-		if (settingsApp != null)
-		{
-			Logger.instance().info("SqlDatabaseIO Overriding Decodes Settings with properties in "
-				+ "Process Record '" + settingsApp + "'");
-			LoadingAppDAI loadingAppDAO = makeLoadingAppDAO();
-			try
-			{
-				CompAppInfo cai = loadingAppDAO.getComputationApp(settingsApp);
-				PropertiesUtil.loadFromProps(DecodesSettings.instance(), cai.getProperties());
-			}
-			catch (DbIoException ex)
-			{
-				Logger.instance().warning("Cannot load settings from app '" + settingsApp + "': " + ex);
-			}
-			catch (NoSuchObjectException ex)
-			{
-				Logger.instance().warning("Cannot load settings from non-existent app '" 
-					+ settingsApp + "': " + ex);
+						q = "ALTER SESSION SET nls_date_format = 'yyyy-mm-dd hh24:mi:ss'";
+						Logger.instance().info(q);
+						st.execute(q);
+
+						q = "ALTER SESSION SET nls_timestamp_format = 'yyyy-mm-dd hh24:mi:ss'";
+						Logger.instance().info(q);
+						st.execute(q);
+
+						q = "ALTER SESSION SET nls_timestamp_tz_format = 'yyyy-mm-dd hh24:mi:ss'";
+						Logger.instance().info(q);
+						st.execute(q);
+
+						Logger.instance().info("DECODES IF Connected to TSDB Version " + tsdbVersion);
+					}
+					catch(SQLException ex)
+					{
+						String msg = "Error in '" + q + "': " + ex
+							+ " -- will proceed anyway.";
+						Logger.instance().failure(msg + " " + ex);
+					}
+
+					cgl.setLoginSuccess(true);
+					Logger.instance().info(module + 
```

Comment:
This logged message contains redundancies with previous "Connect to DECODES CWMS Database" message